extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -251,9 +251,11 @@
   private CircularBuffer<Throwable> savedParseExceptions;
   private IngestionState ingestionState;
 
-  private TaskMetricsGetter metricsGetter;
   private String errorMsg;
 
+  private RowIngestionMetersFactory rowIngestionMetersFactory;
```

Comment:
can both of these be final and handled in the constructor?

Reply:
moved to constructor